egs/swbd/s5c/local/run_asr_segmentation.sh

```diff
@@ -40,8 +42,8 @@ affix=_1a
 stage=-1
 nj=80
 
-. ./path.sh
-. ./cmd.sh
+. path.sh
```

Comment:
The standard now is to use `. ./cmd.sh` and `if [ -f ./path.sh ]; then . ./path.sh; fi`
Better to change it to this in all files.